src/views/layouts/layout.hbs

```diff
@@ -7,6 +7,8 @@
     <body>
         {{> header}}
         {{{content}}}
-        {{> footer}}
+        <div class="container-fluid text-center">
```

Comment:
In this PR the header has the container as part of it's code, perhaps the footer should do the same for consistency?